tools/restart_network.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/sh
 
-function warning {
-    echo "`basename $0`: Warning: ${@:1}" 1>&2;
+warning() {
+    printf "%s: Warning: %s\n" "$(basename $0)" "$*" 1>&2
```

Comment:
$* is no replacement for "$@" see:
https://github.com/koalaman/shellcheck/wiki/SC2048